stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -743,17 +754,15 @@ public func _graph<State : _TensorArrayProtocolEnhanced,
                            "Should not be in tracing mode already!")
 
   // Switch to tracing mode.
-  let inputValueCount = Int(state._tensorHandleCount) + Data._typeList.count
-  _RuntimeConfig.traceState = .tracing(
-    TraceContext(inputValueCount: inputValueCount))
+  let dtypes = state._dtypes() + (Data._typeList.map { $0.cDataType })
```

Comment:
Parentheses are redundant.

```suggestion
  let dtypes = state._dtypes() + Data._typeList.map { $0.cDataType }
```